src/shared/site-config/SocialShare.js

```diff
@@ -7,31 +7,37 @@ const SocialShare = () => {
       <li className="share-item">
         <a
           href={`https://www.facebook.com/sharer.php?u=${window.location.href}`}
+          className="share-link"
```

Comment:
Technically, for proper BEM naming, these should be: `share__item` and `share__link`.